client/me/purchases/components/payment-method-sidebar/index.jsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import CardHeading from 'calypso/components/card-heading';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function PaymentMethodSidebar( { purchase } ) {
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+
+	return (
+		<React.Fragment>
+			{ renderMainCard( purchase, translate, moment ) }
+
+			<Card className="payment-method-sidebar__security-card">
+				<CardHeading
+					tagName="h1"
+					size={ 16 }
+					isBold={ true }
+					className="payment-method-sidebar__title"
+				>
+					{ translate( 'Security' ) }
+				</CardHeading>
+				<p className="payment-method-sidebar__paragraph">
+					{ translate(
+						'Your payment details are safe. We will transfer them over an encrypted connection and store them on a certified payment processing server.'
+					) }
+				</p>
+			</Card>
+		</React.Fragment>
+	);
+}
+
+function renderMainCard( purchase, translate, moment ) {
+	if ( purchase ) {
+		const purchaseMessaging = purchase.renewDate
+			? translate( 'Next payment on %s', { args: moment( purchase.renewDate ).format( 'LL' ) } )
+			: translate( 'Expires on %s', { args: moment( purchase.expiryDate ).format( 'LL' ) } );
+
+		return (
+			<Card className="payment-method-sidebar__details-card">
+				<CardHeading
+					tagName="h1"
+					size={ 16 }
+					isBold={ true }
+					className="payment-method-sidebar__title"
+				>
+					{ translate( 'Purchase details' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Purchase Details' )` *ES Score: 10*

Reply:
Updated string to use uppercase D.